pandas/tests/test_multilevel.py

```diff
@@ -48,31 +48,28 @@ def test_reindex_level(self, multiindex_year_month_day_dataframe_random_data):
         expected = ymd.groupby(level="month").transform(np.sum).T
         tm.assert_frame_equal(result, expected)
 
-    def test_binops_level(self, multiindex_year_month_day_dataframe_random_data):
+    @pytest.mark.parametrize("opname", ["sub", "add", "mul", "div"])
```

Comment:
i think we parameterize these ops?

Reply:
Looks like we have a fixture with the dunder versions but not the functions versions. Plus I don't think all the fixture versions are supported here.